notesapi/v1/tests/test_views.py

```diff
@@ -274,121 +355,136 @@ def test_delete_notfound(self):
         response = self.client.delete(url, self.headers)
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND, "response should be 404 NOT FOUND")
 
+
+class AnnotationSearchlViewTests(BaseAnnotationViewTests):
```

Comment:
could we add a test that ensures that recently-deleted notes don't turn up in search results?